style.css

```
@@ -1,3 +1,13 @@
+@font-face {
+    font-family: "devicon";
```

Stylelint (17.7.0)

[error] Expected no quotes around "devicon" (font-family-name-quotes)

(font-family-name-quotes)